packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+import { Heading } from "../Heading";
+import { Badge } from "../Badge";
+import { Icon } from "../Icon";
+
+export enum InteractiveElementType {
+  Card = "card",
+}
+
+export type UtilityCardProps = {
+  /** Sets the card image source */
+  imageAlt: string;
+  /** Sets the card image alt */
+  imageSrc: string;
+  /** Sets the title to be rendered as h2 */
+  title: string;
+  /** Sets the type of the clickable element */
+  interactiveElementType?: InteractiveElementType;
+  /** Sets the href to be added to the interactive element */
+  href?: string;
+  /** Text describing the kind of service provided */
+  serviceTag: string;
+  /** Text describing the current status of the service as a badge */
+  status?: string;
+  /** Used by StyledComponents to render the component as a specific tag. If href is passed it'll be rendered as "a" */
+  as?: React.ComponentProps<typeof Box.div>["as"];
+  /** Callback function to be used to invoke onClicks */
+  onClick?: React.MouseEventHandler;
+};
+
+const cardBg = {
+  _: "colorBackgroundWeakest",
+  focus: "colorBackgroundWeakest",
+};
+
+const interactiveBg = {
+  active: "colorBackgroundWeak",
+  hover: "colorBackgroundWeak",
+  ...cardBg,
+};
+
+export const UtilityCard: React.FC<UtilityCardProps> = ({
+  as = "div",
+  imageAlt,
+  imageSrc,
+  interactiveElementType,
+  href,
+  title,
+  serviceTag,
+  status,
+  onClick,
+}) => {
+  const isCardInteractive =
+    InteractiveElementType.Card === interactiveElementType;
+
+  const interactiveElementProps = {
+    href,
+    onClick,
+    target: "_blank",
+    rel: "noopener noreferrer",
+    cursor: "pointer",
+  };
+
+  return (
+    <Box.div
+      as={isCardInteractive ? "a" : as}
+      backgroundColor={isCardInteractive ? interactiveBg : cardBg}
+      borderRadius="borderRadius30"
+      display="flex"
+      flexDirection={{ _: "column", lg: "row" }}
```

Comment:
Shouldn't this be always `row`? We have this with the current code: 

![image](https://github.com/Localitos/pluto/assets/5320963/5341c9d9-a7ba-448e-bdf3-84418e6eaa1e)

If we want to have something like this, it's necessary to have a boolean prop to control when there is more than one card being used.